### PR TITLE
Paymark online eftpos

### PR DESCRIPTION
**Pull request by click-paymark**
** Checklist **
- [x ] All icons include `width="38" height="24"` in their attributes
- [x ] All icons have solid white backgrounds
- [x ] All icons have a a `viewBox` attribute
- [x ] All fonts have been converted to paths
- [x ] All icons have a corresponding entry in `db/payment_icons.yml`
- [x ] I am confident that all icons are clear and easy to read/understand

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: db/payment_icons.yml [hunk 1]
```diff
@@ -242,3 +242,7 @@
   name: dailyyamazaki
   label: Daily Yamazaki
   group: convenience_stores
+-
+  name: paymark_online_eftpos
+  label: Paymark Online Eftpos
+  group: bank_transfers
```
Comment (AnthonyClark) [inline db/payment_icons.yml]: Please add the newline at the end of the file

Comment (click-paymark) [inline db/payment_icons.yml]: I added the new line, let me know if you need anything else.

Comment (mirandayoung) [inline db/payment_icons.yml]: Can you please give me an update on whether this logo has been accepted?

Comment (AnthonyClark) [inline db/payment_icons.yml]: Looks good but there's a merge conflict again that'll need another rebase.

Comment (click-paymark) [inline db/payment_icons.yml]: @AnthonyClark I pushed an update fixing the conflict, let me know if you need anything else.

**Pull request merged**